frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -288,6 +291,13 @@ describe("scenarios > collection_defaults", () => {
         cy.signIn("nocollection");
       });
 
+      it.skip("should not render collections in items list if user doesn't have collection access (metabase#16555)", () => {
+        cy.visit("/collection/root");
+        cy.get(".ContentTable")
```

Comment:
@kulyk - this will fail if the collection is empty, right?
I've just realized that. Will have to update the repro to check for "Nothing to see yet."

Reply:
Yes, exactly